### PR TITLE
Fixed client muting

### PR DESCRIPTION
**Pull request by Najsr**
As mentioned in https://github.com/ReSpeak/TS3Hook/pull/18#issuecomment-381760244

### CODE REVIEW

**Review by Bluscream — CHANGES_REQUESTED** [review]
After compiling and trying i only get
```
<23:47:24> Trying to connect to server on XX.XX.XX.XX:XXXX
<23:47:25> undefined error
```
```
[OUT] clientinit client_nickname=bluÛºé client_version=3.?.?\s[Build:\s5680278000] client_platform=Windows                                                                                                                                                                                                                                                                                                                                                                                      
[IN ] error id=1 msg=undefined\serror
```

**Comment by Najsr** [comment]
Should be ready to merge.

File: TS3Hook/dllmain.cpp [hunk 1]
```diff
@@ -311,7 +311,7 @@ void STD_DECL log_out_packet(char* packet, int length)
 			in_str.insert(client_version_sign + 20, clientver[2]);
 		}
 		if (!clientver[1].empty()) {
-			in_str.erase(client_platform + 16, (client_input_hardware - client_platform - 17));
+			in_str.erase(client_platform + 16, (client_input_muted - client_platform - 17));
```
Comment (Bluscream) [inline TS3Hook/dllmain.cpp]: That line does cause the `undefined error` for me. The rest works fine

**Comment by Najsr** [comment]
`[OUT] clientinit client_nickname=Najsr client_version=3.1.7\s[Build:\s1513163251] client_platform=Linux client_input_muted=1 client_output_muted=1 client_input_hardware=1 client_output_hardware=1 client_default_channel client_default_channel_password client_server_password client_meta_data client_version_sign=\/j5TZqPuOU8yMYPdGehvijYvU74KefRrKO5sgTUrkpeslNFiy4XfU7quKW0diLHQoPQn1t3KArdfzOAMk8dlAg== client_key_offset=000 client_nickname_phonetic client_default_token client_badges=Overwolf=0 hwid=XXX`  Works fine for me.

**Comment by Bluscream** [comment]
@Najsr can you give me a compiled win_x64.dll from you?

**Comment by Najsr** [comment]
Give it a try now, should be fixed

**Comment by Bluscream** [comment]
Thanks, i manually merged it in 5866b7e86f9aede3a161d26d879dbd74e4559f70 due to local changes :)

**Pull request closed without merging**